docs/api/stories.md

```diff
@@ -0,0 +1,497 @@
+---
+title: 'Stories'
+---
+
+### Component Story Format (CSF)
+
+Storybook's Component Story Format (CSF) is the recommended way to [write stories](../writing-stories/introduction) since Storybook 5.2. [Read the announcement](https://medium.com/storybookjs/component-story-format-66f4c32366df) to learn more about how it came to be.
+
+<div style="background-color:#F8FAFC">
+TODO: add advanced readme link and if stories of api needs to be maintained)
+</div>
+
+> If you are writing stories in the older `storiesOf()` syntax, you can find documentation in an [advanced README].
+
+In CSF, stories and component metadata are defined as ES Modules. Every component story file consists of a required **default export** and one or more **named exports**.
+
+CSF is supported in all frameworks except React Native, where you should use the [storiesOf API](../formats/storiesof-api/index) instead.
```

Comment:
This should link to the advanced README too

Reply:
Added the link, let me know if it's ok.